docs/tempo/website/getting-started/tempo-in-grafana.md

```diff
@@ -0,0 +1,48 @@
+5. [View json file](#view-json-file)
+
+# View trace by ID
+The most basic functionality is to visualize a trace using its ID.  Select the Trace ID tab and enter the ID to view it. This functionality is enabled by default and is available in all versions of Grafana.
+<p align="center"><img src="../grafana-query.png" alt="View trace by ID"></p>
+
+# Log Search
+Traces can be discovered by searching logs for entries containing trace IDs.  This is most useful when your application also logs relevant information about the trace that can also be searched, such as HTTP status code, customer ID, etc.  This feature requires Grafana 7.5 or later, with a linked Loki data source, and a traceID derived field.
+
+<p align="center"><img src="../log-search.png" alt="Log Search"></p>
+
+
+# Tempo Search
+<span style="background-color:#f3f973;">This experimental feature is disabled by default. See below for more information on how to enable.</span>
+
+Tempo includes native search of recent traces.  Traces can be searched for data originating from a specific service, duration range, and span and process-level attributes included in your application's instrumentation, such as HTTP status code, customer ID, etc.  Currently only search of traces at the ingesters is supported. By default the ingesters store the last 15 minutes.
+
+Enabling this feature requires the following:
+1. Run the Tempo 1.2 or the latest pre-release and enable search in the YAML config. For more information see the [Tempo configuration documentation](../../configuration#search).
+2. Run Grafana 8.2 or the latest pre-release and enable the `tempoSearch` [feature toggle](https://github.com/grafana/tempo/blob/main/example/docker-compose/tempo-search/grafana.ini).
+
+<p align="center"><img src="../tempo-search.png" alt="Tempo Search"></p>
+
+# Service Map
+<span style="background-color:#f3f973;">This experimental feature is disabled by default. See below for more information on how to enable.</span>
+
+A service map can be generated from tracing data and viewed in Grafana by using the [span metrics feature](https://grafana.com/docs/agent/latest/configuration/traces-config/) of Grafana Agent.  This requires that Grafana Agent runs as part of your [tracing pipeline](..).
+
+Enabling this feature requires the following:
+1. Run Grafana Agent v0.14.0 or later and configure it to generate span metrics.
+2. Grafana
+    1. Run Grafana 8.2 or later
+    2. Enable the `tempoServiceGraph` feature toggle.
+    3. In the Tempo datasource, configure a linked metrics data source (i.e. Prometheus) containing the span metrics generated by the Agent.
+
+# View json file
+A local json file containing a trace can be uploaded and viewed in the Grafana UI.  This is useful in cases where access to the original Tempo datasource is limited, or for preserving traces outside of Tempo.  The json data can be downloaded via the Tempo API or the Inspector panel while viewing the trace in Grafana.
```

Comment:
```suggestion
## View JSON file
A local JSON file containing a trace can be uploaded and viewed in the Grafana UI.  This is useful in cases where access to the original Tempo datasource is limited, or for preserving traces outside of Tempo.  The JSON data can be downloaded via the Tempo API or viewed in the Inspector panel while viewing the trace in Grafana.
```

Reply:
The inspector panel has a dedicated `Download Traces` button, so it can be used to view or download.  How about something like `downloaded via the Tempo API or clicking Download traces in the Inspector panel while viewing the trace in Grafana`?